<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>720</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>تنظیمات</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>691</width>
     <height>701</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="settinglayout" stretch="2,1,1">
    <item>
     <widget class="QGroupBox" name="colorgroupbox">
      <property name="title">
       <string>تنظیم رنگ</string>
      </property>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>671</width>
         <height>311</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="colorsettingglayout" rowstretch="0,0,0,0,0,0,0,0,0,0,0,0">
        <property name="verticalSpacing">
         <number>8</number>
        </property>
        <item row="3" column="0">
         <widget class="QLabel" name="colorclock_l_r">
          <property name="text">
           <string>شدت روشنایی قرمز:</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QSlider" name="coloraround_s_a">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="11" column="1">
         <widget class="QSlider" name="coloraround_s_b">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSlider" name="colorclock_s_r">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QSlider" name="colorclock_s_g">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="colorclock_l_b">
          <property name="text">
           <string>شدت روشنایی رنگ آبی:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSlider" name="colorclock_s_a">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="11" column="2">
         <widget class="QLabel" name="coloraround_v_b">
          <property name="text">
           <string>۰</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="coloraround_l_g">
          <property name="text">
           <string>شدت روشنایی سبز:</string>
          </property>
         </widget>
        </item>
        <item row="10" column="2">
         <widget class="QLabel" name="coloraround_v_g">
          <property name="text">
           <string>۰</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLabel" name="colorclock_v_g">
          <property name="text">
           <string>۰</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="colorclock_v_a">
          <property name="text">
           <string>۰</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QSlider" name="coloraround_s_r">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="QSlider" name="coloraround_s_g">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="coloraround_l_r">
          <property name="text">
           <string>شدت روشنایی قرمز:</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="coloraround_l_a">
          <property name="text">
           <string>تنظیم محو شدن دایره دور:</string>
          </property>
         </widget>
        </item>
        <item row="8" column="2">
         <widget class="QLabel" name="coloraround_v_a">
          <property name="text">
           <string>۰</string>
          </property>
         </widget>
        </item>
        <item row="9" column="2">
         <widget class="QLabel" name="coloraround_v_r">
          <property name="text">
           <string>۰</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="colorclock_l_g">
          <property name="text">
           <string>شدت روشنایی رنگ سبز:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QLabel" name="colorclock_v_b">
          <property name="text">
           <string>۰</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="colorclock_v_r">
          <property name="text">
           <string>۰</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QSlider" name="colorclock_s_b">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="colorclock_l_a">
          <property name="text">
           <string>تنطیم محو شدن پس‌زمینه:</string>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QLabel" name="coloraround_l_b">
          <property name="text">
           <string>شدت روشنایی آبی:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="fontgroupbox">
      <property name="title">
       <string>تنظیم قلم</string>
      </property>
      <widget class="QWidget" name="gridLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>39</y>
         <width>671</width>
         <height>121</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="fontsettingglayout" columnstretch="0,0,0,2,2,1">
        <item row="0" column="5">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>قلم رویداد:</string>
          </property>
          <widget class="QWidget" name="horizontalLayoutWidget_3">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>30</y>
             <width>131</width>
             <height>71</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="eventfonthlayout">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <item>
             <widget class="QFontComboBox" name="eventfontname">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>1</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777204</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="eventfont_size">
              <property name="maximum">
               <number>500</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QGroupBox" name="gbaox_datefont">
          <property name="title">
           <string>قلم تاریخ:</string>
          </property>
          <widget class="QWidget" name="horizontalLayoutWidget_2">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>251</width>
             <height>71</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="datefonthlayout">
            <item>
             <widget class="QFontComboBox" name="datefontname"/>
            </item>
            <item>
             <widget class="QSpinBox" name="datefont_size">
              <property name="maximum">
               <number>500</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="datefontcolor_b">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QGroupBox" name="gbox_clockfont">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="title">
           <string>قلم ساعت:</string>
          </property>
          <widget class="QWidget" name="horizontalLayoutWidget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>30</y>
             <width>261</width>
             <height>71</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="clockfonthlayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <item>
             <widget class="QFontComboBox" name="clockfontname"/>
            </item>
            <item>
             <widget class="QSpinBox" name="clockfont_size">
              <property name="maximum">
               <number>500</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="clockfontcolor_b">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="othersettingsglayout" rowstretch="0,0" columnstretch="0,0">
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1">
        <item>
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QWidget" name="verticalLayoutWidget_3">
           <property name="geometry">
            <rect>
             <x>9</x>
             <y>9</y>
             <width>441</width>
             <height>161</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,1,0,1">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="clockwidth_l">
                <property name="text">
                 <string>اندازه ساعت:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="clockwidth_s">
                <property name="minimum">
                 <number>300</number>
                </property>
                <property name="maximum">
                 <number>800</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="clockwisth_v">
                <property name="text">
                 <string>۳۰۰</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="Line" name="line_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QPushButton" name="updater_b">
                <property name="text">
                 <string>به‌روزرسانی رویدادها</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QProgressBar" name="updater_pb">
                <property name="value">
                 <number>24</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="Line" name="line">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QPushButton" name="aboutme_b">
                <property name="text">
                 <string>درباره</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="clockseconds_b">
          <property name="title">
           <string>حلقه ثانیه شمار:</string>
          </property>
          <widget class="QWidget" name="verticalLayoutWidget_2">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>211</width>
             <height>159</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="seconds_vl">
            <item>
             <widget class="QCheckBox" name="secondshow">
              <property name="text">
               <string>ثانیه شمار:</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,0" columnstretch="1,2">
              <property name="spacing">
               <number>2</number>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="secondcolor_l">
                <property name="text">
                 <string>ثانیه شمار:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QPushButton" name="secondcolor_b">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="secondscolor_l">
                <property name="text">
                 <string>حلقه:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QPushButton" name="secondscolor_b">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
